File: Java/src/main/java/workshops/testingil/dirtytests/exercise/CalculatorParams.java
package workshops.testingil.dirtytests.exercise;

public class CalculatorParams {

	int firstArg;
	int secondArg;
	int thirdArg;
	OperationType op1;
	OperationType op2;

	public void setFirst(int arg) {
		this.firstArg = arg;
	}

	public void setSecond(int arg) {
		this.secondArg = arg;
	}

	public void setThird(int arg) {
		this.thirdArg = arg;
	}

	public void setfirstOp(OperationType op) {
		this.op1 = op;
	}

	public void setSecondOp(OperationType op) {
		this.op2 = op;
	}

}

File: JS/DirtyTestsJS/exercise/CalculatorParams.ts
class CalculatorParams {
    firstArg: number;
    secondArg: number;
    thirdArg: number;
    op1: OperationType;
    op2: OperationType;

    setFirst(arg: number) {
        this.firstArg = arg;
    }

    setSecond(arg: number) {
        this.secondArg = arg;
    }

    setThird(arg: number) {
        this.thirdArg = arg;
    }

    setfirstOp(op: OperationType) {
        this.op1 = op;
    }

    setSecondOp(op: OperationType) {
        this.op2 = op;
    }

}